### DISCUSSION TITLE
Creating a .NET Cron Job for Kubernetes | benbrougher.tech

### DISCUSSION THREAD

**Discussion by giscus** [moutansos/benbrougher-tech#49] [Comments]
# Creating a .NET Cron Job for Kubernetes | benbrougher.tech

How to create a .NET 7 Cron Job with C# running in a Kubernetes Cluster

https://benbrougher.tech/posts/kubernetes-cron-job/

**Comment by Dawid1988** [comment]
I gott this message
error: error validating "kube.yml": error validating data: apiVersion not set; if you choose to ignore these errors, turn validation off with --validate=false
